checkbox-ng/plainbox/impl/config.py

```diff
@@ -250,6 +256,17 @@ def from_ini_file(cls, ini_file, origin):
                     ).format(var_name)
                     cfg.notice_problem(problem)
                 continue
+
+            deprecated_section_names = {
+                "daemon": "agent"
+            }
```

Comment:
what do you mean by the uppercase one? you mean add "DAEMON": "AGENT" in the dict?
I thought since we're keeping the master & slave as workable deprecated terms, we don't need to change file name, no?

Reply:
I mean that you have a constant in the same class named `DEPRECATED_SECTION_NAMES`

As for the module names, keeping the old one workable does not mean keeping them everywhere in the codebase, the backward compatibility (imo, lets talk about this) should be limited to the cli aliases, everything else should be uniform after this PR

Reply:
I completely removed `master` and `slave` from the codebase, only comments indicating that they have been deprecated are retained.

I've tested with tox and metabox local & remote(local is all passed, remote failed, the reason is as Massimiliano said.)